### ISSUE TITLE
Update documentation and create a website for the project

### ISSUE THREAD

**Issue by daynin** [vishesh/racketscript#210]
Now documentation looks a bit outdated, so I think we should try to update it. And as an addition would be great to create a website for the project with a short description section, a "getting started" section, and so on.

**Comment by daynin** [comment]
I think I can do it. At least I can try :) But maybe as far as I'm new here I can notice some gaps in documentation which can be useful

**Comment by vishesh** [comment]
Thanks! Go ahead!! Looking forward to the PR.

**Comment by daynin** [comment]
It can take some time because I'm trying to implement a "real world" example and to use some modern tools like vitejs and esbuild to provide better DX. Maybe this attempt will produce some new PRs or issues.

**Comment by vishesh** [comment]
Cool. Would love to hear what you have in mind. Our goal in general is to avoid JavaScript if possible[1][2] from runtime and examples we show, so would be interested making some form of lightweight binding on top of it? I think our FFI can improve quite a bit from lesson learned from doing such exercises, even such exercises are simple.

Being in Racket ecosystem, I think we'll prefer the documentation to be in [Scribble](https://docs.racket-lang.org/scribble/).

[1] We build RacketScript Playground that way
[2] Our JavaScript runtime used to be bigger than it is until we started moving to RacketScript itself in `racketscript/boot` language. E.g. https://github.com/vishesh/racketscript/blob/master/racketscript-compiler/racketscript/compiler/runtime/kernel.rkt

**Comment by daynin** [comment]
Yeah, of course. I want to implement some react app (maybe something like this https://todomvc.com/examples/react/#/) using modern build tools. It can be useful just to show that people can write modern apps in RacketScript. Now I'm using vitejs (to build the app using ESM) and experimenting with esbuild to provide fast compiling with a small output bundle.

> Our goal in general is to avoid JavaScript if possible[1][2] from runtime and examples we show

This app (which I'm developing right now) will be written completely in RacketScript.

> So would you be interested making some form of lightweight binding on top of it? I think our FFI can improve quite a bit from lesson learned from doing such exercises, even such exercises are simple.

Yes, of course. I'll try to use all existing features in RacketScript to achieve my goal, but If it won't be possible I'll open some issues to discuss improvements (if they will be big enough) or just will improve/add something to make it possible (if the issues will be simple).

And at the end of developing this app, I hope we'll get a real world example, highlighted issues in the compliers, some DX improvements (maybe) and a website about RacketScript with syntax examples, getting started information, and a bunch of useful links (to documentation, readme and so on). So that's the plan.

**Comment by daynin** [comment]
> Being in Racket ecosystem, I think we should use Scribble for docs.

I agree. I'll add only basic documentation to a website and try to use scribble for writing detailed documentation.

**Comment by daynin** [comment]
A small update.

I wrote a wrapper for React https://github.com/rackt-org/rackt. And add some examples here https://rackt-org.github.io/.

And now I'm gonna start to work on a website for RacketScript itself.

P.S.
I invited you (@vishesh and @stchang) to [Rackt's organization](https://github.com/rackt-org). Feel free to join it if you want.

**Comment by stchang** [comment]
> I wrote a wrapper for React https://github.com/rackt-org/rackt. And add some examples here https://rackt-org.github.io/.

Awesome! I was just thinking that I wanted something like this when I was trying to update the playground app

**Comment by stchang** [comment]
Might be cool to add a few macros to show how useful they can be :)

**Comment by daynin** [comment]
Thank you! I think we can add some examples with Rackt to the playground a little bit later.

**Comment by vishesh** [comment]
@daynin : Awesome work. It looks great, and fills a huge gap we had. Also feel free to suggest any changes or opinion on what the interop library should be like. I know RacketScript FFI isn't great and we'll have to make it better or more organized and consistent at the very least. (https://github.com/vishesh/racketscript/issues/237)

**Comment by daynin** [comment]
@vishesh thank you!
